src/test/java/uk/gov/hmcts/reform/sscs/callback/handlers/FilterNotificationsEventsHandlerTest.java

```diff
@@ -29,165 +34,217 @@
+    @Parameters({
+        "ADJOURNED_NOTIFICATION",
+        "ADMIN_APPEAL_WITHDRAWN",
+        "APPEAL_DORMANT_NOTIFICATION",
+        "APPEAL_LAPSED_NOTIFICATION",
+        "APPEAL_RECEIVED_NOTIFICATION",
+        "APPEAL_WITHDRAWN_NOTIFICATION",
+        "DECISION_ISSUED",
+        "DIRECTION_ISSUED",
+        "DIRECTION_ISSUED_WELSH",
+        "DECISION_ISSUED_WELSH",
+        "REVIEW_CONFIDENTIALITY_REQUEST",
+        "JOINT_PARTY_ADDED",
+        "DWP_APPEAL_LAPSED_NOTIFICATION",
+        "DWP_RESPONSE_RECEIVED_NOTIFICATION",
+        "DWP_UPLOAD_RESPONSE_NOTIFICATION",
+        "EVIDENCE_RECEIVED_NOTIFICATION",
+        "ISSUE_ADJOURNMENT_NOTICE",
+        "ISSUE_FINAL_DECISION",
+        "ISSUE_FINAL_DECISION_WELSH",
+        "NON_COMPLIANT_NOTIFICATION",
+        "DRAFT_TO_NON_COMPLIANT_NOTIFICATION",
+        "REISSUE_DOCUMENT",
+        "PROCESS_AUDIO_VIDEO",
+        "PROCESS_AUDIO_VIDEO_WELSH",
+        "ACTION_HEARING_RECORDING_REQUEST",
+        "ACTION_POSTPONEMENT_REQUEST_WELSH",
+        "REQUEST_INFO_INCOMPLETE",
+        "RESEND_APPEAL_CREATED_NOTIFICATION",
+        "STRUCK_OUT",
+        "SUBSCRIPTION_UPDATED_NOTIFICATION",
+        "VALID_APPEAL_CREATED",
+        "DRAFT_TO_VALID_APPEAL_CREATED",
+        "DIRECTION_ISSUED_WELSH",
         "UPDATE_OTHER_PARTY_DATA"
     })
     public void willHandleEvents(NotificationEventType notificationEventType) {
-        SscsCaseDataWrapper callback = SscsCaseDataWrapper.builder().notificationEventType(notificationEventType).build();
-        assertTrue(handler.canHandle(callback));
-        handler.handle(callback);
-        verify(notificationService).manageNotificationAndSubscription(eq(new CcdNotificationWrapper(callback)), eq(false));
-        verifyNoInteractions(retryNotificationService);
+        callback.setNotificationEventType(notificationEventType);
+
+        willHandle(callback);
     }
 
     @Test
     @Parameters({"DO_NOT_SEND", "SYA_APPEAL_CREATED_NOTIFICATION", "POSTPONEMENT_NOTIFICATION", "null"})
     public void willNotHandleEvents(@Nullable NotificationEventType notificationEventType) {
-        SscsCaseDataWrapper callback = SscsCaseDataWrapper.builder().notificationEventType(notificationEventType).build();
-        assertFalse(handler.canHandle(callback));
+        callback.setNotificationEventType(notificationEventType);
+
+        wontHandle(callback);
     }
 
-    @Test(expected = IllegalStateException.class)
     @Parameters({"DO_NOT_SEND", "SYA_APPEAL_CREATED_NOTIFICATION"})
     public void willThrowExceptionIfTriesToHandleEvents(NotificationEventType notificationEventType) {
-        SscsCaseDataWrapper callback = SscsCaseDataWrapper.builder().notificationEventType(notificationEventType).build();
-        handler.handle(callback);
+        callback.setNotificationEventType(notificationEventType);
+
+        wontHandle(callback);
```

Comment:
maybe change it to willNotHandle(). Looks better and matches the willHandle a bit better